lib/cli/watch-run.js

```diff
@@ -85,10 +85,17 @@ const createRerunner = (mocha, beforeRun) => {
 
   let rerunScheduled = false;
 
+  let defaultGlobals = null;
```

Comment:
Can we use another name? Because I feel it is not "default".

Reply:
how about "prevGlobals" or "memoGlobals"?